From 4f43f46b1aa71a06aafaebfbb0eed03a0e4799c6 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 14 Nov 2016 18:43:57 +0200
Subject: [PATCH 1095/2241] dt-bindings: net: add SFI, XFI, XAUI, RXAUI and KR
 phy-modes to ethernet.txt

- update ethernet.txt net bindings with SFI, XFI, XAUI, RXAUI and
  KR phy-modes.

Change-Id: If433d03a8f80237649c6b15a4ad082e60172d11d
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33855
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 Documentation/devicetree/bindings/net/ethernet.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/net/ethernet.txt b/Documentation/devicetree/bindings/net/ethernet.txt
index 5d88f37..d78389d 100644
--- a/Documentation/devicetree/bindings/net/ethernet.txt
+++ b/Documentation/devicetree/bindings/net/ethernet.txt
@@ -11,7 +11,8 @@ The following properties are common to the Ethernet controllers:
   the maximum frame size (there's contradiction in ePAPR).
 - phy-mode: string, operation mode of the PHY interface; supported values are
   "mii", "gmii", "sgmii", "qsgmii", "tbi", "rev-mii", "rmii", "rgmii", "rgmii-id",
-  "rgmii-rxid", "rgmii-txid", "rtbi", "smii", "xgmii"; this is now a de-facto
+  "rgmii-rxid", "rgmii-txid", "rtbi", "smii", "xgmii", "xaui", "rxaui", "sfi"
+  "xfi", "kr"; this is now a de-facto
   standard property;
 - phy-connection-type: the same as "phy-mode" property but described in ePAPR;
 - phy-handle: phandle, specifies a reference to a node representing a PHY
-- 
2.7.4

